client/components/section-nav/README.md

```diff
@@ -29,15 +29,27 @@ export default class extends React.Component {
 		return (
 			<SectionNav selectedText={ sectionNavSelectedText }>
 				<NavTabs label="Status" selectedText="Published">
-					<NavItem path="/posts" selected={ true }>Published</NavItem>
-					<NavItem path="/posts/drafts" selected={ false }>Drafts</NavItem>
-					<NavItem path="/posts/scheduled" selected={ false }>Scheduled</NavItem>
-					<NavItem path="/posts/trashed" selected={ false }>Trashed</NavItem>
+					<NavItem path="/posts" selected={ true }>
```

Comment:
I'd love to write a codemod for those.

```suggestion
					<NavItem path="/posts" selected>
```

Reply:
Maybe we can just enable https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

Reply:
I don't see why not 😉